src/source/Signaling/LwsApiCalls.c

```diff
@@ -1117,6 +1081,8 @@ STATUS deleteChannelLws(PSignalingClient pSignalingClient, UINT64 time)
 {
     ENTERS();
     STATUS retStatus = STATUS_SUCCESS;
+    UNUSED_PARAM(time);
```

Comment:
Why arent we using the time parameter anymore?

Reply:
The caching aggregate should handle the awaiting now. The caching aggregate then calls the LWS routine to perform the operation if the call actually need to be made. In case of delete, we don't do caching.